src/keylessBackup/encryption.ts

```diff
@@ -100,9 +100,17 @@ export function decryptPassphrase(
   encryptedData: string
 ): string {
   const derivedKey = deriveKeyFromKeyShares(keyshare1, keyshare2)
-  const [nonceBase64, encrypted, authTagBase64] = encryptedData.split(':')
+  const [nonceBase64, encrypted, authTagBase64OrSerializedArrayBuffer] = encryptedData.split(':')
   const nonce = Buffer.from(nonceBase64, 'base64')
-  const authTag = Buffer.from(authTagBase64, 'base64')
+  // There was a bug with the initial switch to react-native-quick-crypto,
```

Comment:
just for my understanding, with this hack, we don't need the patch right?

Reply:
The patch is still needed so we don't create encrypted data with the incorrectly serialized authTag, moving forward.

This code here, is to recover from the case someone tried to use Cloud Account Backup with `react-native-quick-crypto` without the patch.
I think this would only be internal devs. But I guess probably only me got into that state, because the crypto operation failed without the `@toruslabs` lib update.
And no public release went out with the broken Cloud Account Backup.

Happy to clarify this more if necessary.